tests/test_default.yml

```diff
@@ -1,6 +0,0 @@
-# SPDX-License-Identifier: BSD-3-Clause
----
-- name: Test executing the role with default parameters
-  hosts: all
-  roles:
-    - linux-system-roles.network
```

Comment:
Tests seem to be gone, there are only deletions, no additions.

Reply:
Thank you, too much rebase fun. This should be fixed now.